spec/slack-ruby-bot/hooks/message_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+class DummyBot < SlackRubyBot::Bot; end
+describe SlackRubyBot::Hooks::Message do
+  let(:server) { SlackRubyBot::Server.new }
+  describe '#child_command_classes' do
+    it 'returns only child command classes' do
+      child_command_classes = server.send(:child_command_classes)
+      expect(child_command_classes).to include(SlackRubyBot::Bot)
```

Comment:
Minor nitpick, a lot of these parenthesis are not needed, you can just do `include SlackRubyBot::Bot`, it's English... errr... I mean Ruby :)